tutorials/optical_tactile_plugin.md

```diff
@@ -53,19 +48,25 @@ The following assumptions have been made when working on this plugin:
 
 - Currently, the plugin's model should be made up of one link, one contact
 sensor and one depth camera.
-- The geometry of the sensor must be a box. Ideally, the size should be around
-20x20x5 mm, but this can be changed.
+- The geometry of the sensor must be a box, because the interpolation of the 
+contacts returned assumes 4 points of input. The visualization of contacts 
+and the `<extended_sensing>` parameter currently make this assumption as well.
+Ideally, the size should be around 20x20x5 mm, but this can be changed. That is
+the approximate size of a real tactile sensor. A bigger one will also work,
+as long as the contact surface fits inside the camera's FOV.
 - The depth camera should be displaced from the model's origin in the negative
-direction of the X axis, so it's not placed inside the membrane. This can be done
-setting the depth camera's `<pose>` accordingly. Check out the [example below](#try-it-out).
-- There should not be a `<visual>` element inside the link, so the depth camera
-isn't facing the link and can return coherent values. The 'membrane' can still
-be visualized setting the `<visualize_sensor>` parameter to true. For more information
-about the parameters, check the [parameters section](#parameters).
+direction of the X axis, so it's not placed inside the contact surface. This can
```

Comment:
It's not clear what frame "X axis" is referring to here. It would be clearer to say what entity this is with respect to, or what surface it is orthogonal to, etc., rather than using the variable.

Reply:
457691b 
I think now should be clear and it should be enough with the explanation and the `Components layout` and `Try it out` sections.